test/src/csg/TestCSGCylindersMeshGenerator.C

```diff
@@ -0,0 +1,87 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "TestCSGCylindersMeshGenerator.h"
+#include "CSGBase.h"
+
+registerMooseObject("MooseTestApp", TestCSGCylindersMeshGenerator);
+
+InputParameters
+TestCSGCylindersMeshGenerator::validParams()
+{
+  InputParameters params = MeshGenerator::validParams();
+
+  params.addRequiredParam<std::vector<Real>>("radii", "list of radii for concentric cylinders");
+  params.addRequiredParam<Real>("height", "cylinder height");
+  params.addRequiredParam<std::vector<Real>>("center", "center point of cylinder");
```

Comment:
```suggestion
  params.addRequiredParam<Point>("center", "center point of cylinder");
```

Reply:
the `CSG[X]Cylinder` method isn't setup to take in a point anyway, so if you are okay with it, I will leave this test input as it is.